textextraction/doc_process_toolkit.py

```diff
@@ -18,18 +18,23 @@ def get_doc_length(doc_text):
     return len(tuple(WORDS.finditer(doc_text)))
 
 
-def check_for_text(doc_path):
+def check_for_text(doc_path, extension):
```

Comment:
I actually run, 2 checks for text. The first is a quick scan to make sure that text even exists in the document. The second occurs after Tika extracts the document text to make sure that the text extraction worked. I figured it would save some Tika processing time.